### PR TITLE
api_modify, api_info: support more fields in existing paths

### LINKED ISSUE #115: api_modify unsupported keys
Hello there, 
I've started to work with this collection to configure routeros hardware, first of all thanks for this great piece of software it's a real pleasure working with it !

##### SUMMARY
I've encountered some keys unsupported by api_modify :
```
- name: ip firewall filter
  community.routeros.api_modify:
    path: ip firewall filter
    data: 
      "{{ firewall_filters }}"

loop item :
        {
            "action": "fasttrack-connection",
            "chain": "forward",
            "connection-state": "established,related",
            "hw-offload": true,
        },

result:
TASK [mikrotik : ip firewall filter] 
FAILED! => {"changed": false, "msg": "Unknown key \"hw-offload\" at index 1."}
```

Same here : 
```
- name: interface ethernet
  community.routeros.api_modify:
    path: interface ethernet
    data: 
      "{{ interfaces_ethernet }}"

loop item: 
        {
            "default-name": "ether5",
            "name": "ether5-test",
            "poe-out": "auto-on"
        },

result:
TASK [mikrotik : interface ethernet] 
FAILED! => {"changed": false, "msg": "Unknown key \"poe-out\" for default-name=\"ether5\"."}
```
I don't use it but I noticed poe-priority key is also missing for "interface ethernet" path

I've added these keys in my local plugins/module_utils/_api_data.py file and these tasks are running happily

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
community.routeros.api_modify

##### ANSIBLE VERSION
```
ansible [core 2.12.9]
  python version = 3.10.7 (main, Sep  7 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-1)]
  jinja version = 3.0.3
  libyaml = True
```

##### COLLECTION VERSION
```
Collection         Version
------------------ -------
community.routeros 2.3.0
```

##### OS / ENVIRONMENT
Mikrotik ROS 7.5 (stable)

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fixes #115.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
api_modify
api_info

### CODE REVIEW

**Comment by felixfontein** [comment]
@therfert if you're interested in reviewing a similar PR... :)

File: plugins/module_utils/_api_data.py [hunk 1]
```diff
@@ -113,16 +113,27 @@ def join_path(path):
...
             'mac-address': KeyInfo(),
+            'mdix-enable': KeyInfo(),
             'mtu': KeyInfo(default=1500),
             'name': KeyInfo(),
             'orig-mac-address': KeyInfo(),
+            'poe-out': KeyInfo(can_disable=True),
+            'poe-priority': KeyInfo(can_disable=True),
+            'poe-voltage': KeyInfo(can_disable=True),
+            'power-cycle-interval': KeyInfo(),
+            'power-cycle-ping-address': KeyInfo(),
```
Comment (therfert) [inline plugins/module_utils/_api_data.py:134]: There is an "unset" command for power-cycle-ping-address, so should we add can_disable=True ? 
```suggestion
            'power-cycle-ping-address': KeyInfo(can_disable=True),
```

File: plugins/module_utils/_api_data.py [hunk 2]
```diff
@@ -113,16 +113,27 @@ def join_path(path):
...
+            'power-cycle-ping-enabled': KeyInfo(),
+            'power-cycle-ping-timeout': KeyInfo(),
```
Comment (therfert) [inline plugins/module_utils/_api_data.py:136]: same as previous
```suggestion
            'power-cycle-ping-timeout': KeyInfo(can_disable=True),
```

**Review by therfert — CHANGES_REQUESTED** [review]

File: plugins/module_utils/_api_data.py (hunk 1 above)
Comment (felixfontein) [reply plugins/module_utils/_api_data.py:134]: Sounds good!

**Review by therfert — APPROVED** [review]
Looks good to me!

**Comment by felixfontein** [comment]
@therfert thanks for reviewing this!

**Pull request merged**